test_runner/batch_others/test_sync_safekeepers_restart.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+import random
+import time
+
+from contextlib import closing
+from multiprocessing import Process, Value
+from fixtures.zenith_fixtures import ZenithEnvBuilder
+from fixtures.log_helper import log
+
+pytest_plugins = ("fixtures.zenith_fixtures")
+
+
+# Test safekeeper sync
+# Ensure that basebackup after safekeepers restart is correct
```

Comment:
Let's comment (and/or change test name) reflecting that it tests creating compute when *previous compute got down while pageserver was lagging behind safekeepers*.

Reply:
Done